great_expectations/self_check/util.py

```diff
@@ -2004,3 +1964,16 @@ def generate_test_table_name(
         [random.choice(string.ascii_letters + string.digits) for _ in range(8)]
     )
     return table_name
+
+
+def _create_bigquery_engine() -> Engine:
+    gcp_project = os.getenv("GE_TEST_BIGQUERY_PROJECT")
+    if not gcp_project:
+        raise ValueError(
+            "Environment Variable GE_TEST_BIGQUERY_PROJECT is required to run expectation tests"
+        )
+    return create_engine(f"bigquery://{gcp_project}/{_bigquery_dataset()}")
+
+
+def _bigquery_dataset() -> str:
+    return os.getenv("GE_TEST_BIGQUERY_DATASET", "test_ci")
```

Comment:
Just confirming that the default value `"test_ci"` is not equivalent to the env var?

Reply:
Shoot I missed this one (yes it's equivalent to the env_var). I'll take out before merging